### DISCUSSION TITLE
Could you please send a debug version to investigate the ingestion stop

### DISCUSSION THREAD

**Discussion by noamlivne** [chuot/rdio-scanner#157] [General]
Hi Chrystian,
Hopefully last issue for this month...
Sorry for all the trouble. 
When you have time, can you please send a "talkative" debug version so I can see what is the error when Rdio-scanner in Windows10, with SDRTrunk and Tailscale stops to ingest the calls all of a sudden. 
Perhaps it is again an issue of the "VPN" thread closure? But no error appears.
Perhaps it is related to https://github.com/chuot/rdio-scanner/issues/156 ?
Thank you in advance.

**Comment by chuot** [comment]
There are a few things that have been fixed in v6.2.3 that could lead to deadlocks or race conditions.

**Reply by noamlivne** [reply]
So far no deadlocks in the new version.
Thank you very much.

**Comment by chuot** [comment]
Hopefully this is the end of a saga. Thanks!
